carapace-server/src/main/java/org/carapaceproxy/server/mapper/StandardEndpointMapper.java

```diff
@@ -75,6 +76,9 @@
     private static final String ACME_CHALLENGE_URI_PATTERN = "/\\.well-known/acme-challenge/";
     private DynamicCertificatesManager dynamicCertificateManger;
 
+    public static final String DEBUGGING_HEADER_ROUTING_PATH = "Routing-Path";
```

Comment:
better to make it configurable
the debug MUST be a name which starts with "X-", which means it is a custom header